src/BinaryPlatforms.jl

```diff
@@ -178,22 +178,22 @@ struct Linux <: Platform
             throw(ArgumentError("Unsupported libc '$libc' for Linux"))
         end
 
-        # Auto-map the `call_abi` to be `eabihf` on armv7l
-        if call_abi === nothing && arch === :armv7l
+        # Auto-map the `call_abi` to be `eabihf` on armv6l/armv7l
+        if call_abi === nothing && arch in (:armv6l, :armv7l)
```

Comment:
You could define a helper function like
```julia
isarm32(arch::Symbol) = arch === :armv6l || arch === :armv7l
```
then replace all of the `arch in (:armv6l, :armv7l)`s with `isarm32(arch)`.